Frontend/CO.CDP.OrganisationApp/Pages/Forms/DynamicFormsPage.cshtml.cs

```diff
@@ -164,6 +169,37 @@ private bool ValidateTextAnswer()
         return true;
     }
 
+    public bool ValidateFileUpload()
+    {
+        var file = Request.Form.Files["fileUpload"];
+        
+        var allowedFileSizeMB = 5;
+        var allowedExtensions = new List<string> { ".pdf", ".docx", ".csv", ".jpg", ".bmp", ".png", ".tif" };
```

Comment:
@rmohammed-goaco @glgoaco @JBaigGoaco should any validation rules be returned as configuration options from the API? Otherwise, we'll need to duplicate validation on the API side.

Reply:
Ideally this should be moved to (refactored) in its own model

Reply:
@rmohammed-goaco As Discussed in the chat please update the PR. Many thanks.